webapp/src/main/resources/application.yml

```diff
@@ -89,6 +89,10 @@ permissionservice:
   endpoint: ${rdw.permission.service}
 
 app:
+  reporting:
+    webapp:
+      url: /
```

Comment:
Ah, it's even more complicated than i knew. I will quietly back out of the room and let the UX folks weigh in.